src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/statusindicator/StatusIndicator.kt

```diff
@@ -47,6 +48,63 @@ class StatusIndicator : Control() {
         }
     var trackFill: Color by trackFillProperty
 
+    val barBorderStyleProperty: StyleableObjectProperty<BorderStrokeStyle> =
+            object: SimpleStyleableObjectProperty<BorderStrokeStyle>(
+                BAR_BORDER,
+                StatusIndicator,
+                "barBorderStyle",
+                BorderStrokeStyle.NONE
+            ){
```

Comment:
do you need the parentheses if there's nothing in them?

Reply:
if i remove the curly braces i get an error from the IDE which says `expecting class body`